lib/src/auth/exception/firebase_auth_exception_code.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:firedart/firedart.dart';
+
+/// Represents a [FirebaseAuthException] code.
+enum FirebaseAuthExceptionCode {
+  /// Indicates whether the Firebase API key is not valid.
+  invalidApiKey,
+
+  /// Indicates whether the user with such email is not found.
```

Comment:
Similar to the above:
```suggestion
  /// An exception code indicating that the user with such email is not found.
```
Please apply to all error codes.